mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageIdDAO.java

```diff
@@ -210,7 +210,7 @@ private PreparedStatement prepareSelectUidRange(Session session) {
         return row
                 .map(this::fromRowToComposedMessageIdWithFlags)
                 .map(Optional::of)
-                .defaultIfEmpty(Optional.empty());
+                .switchIfEmpty(Mono.just(Optional.empty()));
```

Comment:
what is the difference?

Reply:
that's a rebase issue, for sure. (i did the exact reverse change in another PR...)